application/config.json.template

```diff
@@ -89,10 +89,16 @@
     "logInfoChannelWebhook": "<put_your_webhook_here>",
     "logErrorChannelWebhook": "<put_your_webhook_here>",
     "openaiApiKey": "<check pins in #tjbot_discussion for the key>",
-    "sourceCodeBaseUrl": "<https://github.com/<your_account_here>/<your_repo_here>/blob/master/application/src/main/java/>"
+    "sourceCodeBaseUrl": "<https://github.com/<your_account_here>/<your_repo_here>/blob/master/application/src/main/java/>",
     "jshell": {
         "baseUrl": "<put_jshell_rest_api_url_here>",
         "rateLimitWindowSeconds": 10,
         "rateLimitRequestsInWindow": 3
+    },
+    "feature_blacklist": {
+        "normal_feature_blacklist": [
+        ],
+        "special_feature_blacklist": [
+        ]
```

Comment:
redudancy in the names. its clear from context already that this is a feature blacklist. so just `normal` and `special` would be good enough at this point.

that said, from a user point of view, its kinda annoying that this is split - would it be possible to only have a single list? id prefer that then.

Reply:
Fixed, tell me what you think.